Redis.cpp

```diff
@@ -715,6 +715,12 @@ RedisSubscribeResult Redis::startSubscribingNonBlocking(RedisMsgCallback message
   return RedisSubscribeSuccess;
 }
 
+bool Redis::alive()
```

Comment:
```suggestion
bool Redis::ping()
```

Unless absolutely necessary we'd prefer to keep methods that invoke Redis commands named the same as the Redis command they invoke.